ntbs-service/Pages/ValidationModel.cs

```diff
@@ -1,26 +1,41 @@
 using System;
```

Comment:
will rename this file later

Reply:
Having this as a separate class also means we should be able to write tests for it much more easily :) As we can just create a mock page model